tests/network/vmi_multus.go

```diff
@@ -466,6 +466,32 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 				Expect(runSafeCommand(vmiOne, fmt.Sprintf("ip addr show eth0 | grep %s\n", interfacesByName["default"].MAC))).To(Succeed())
 				Expect(runSafeCommand(vmiOne, fmt.Sprintf("ip addr show eth1 | grep %s\n", interfacesByName["linux-bridge"].MAC))).To(Succeed())
 			})
+
+			It("should have the correct MTU on the secondary interface with no dhcp server", func() {
+				const mtuPath = "/sys/class/net/eth1/mtu"
+				getPodInterfaceMtu := func(vmi *v1.VirtualMachineInstance) string {
+					vmiPod := tests.GetRunningPodByVirtualMachineInstance(vmi, vmi.Namespace)
+					output, err := tests.ExecuteCommandOnPod(
+						virtClient,
+						vmiPod,
+						"compute",
+						[]string{"cat", mtuPath},
+					)
+					ExpectWithOffset(1, err).ToNot(HaveOccurred())
+
+					return strings.TrimSuffix(output, "\n")
+				}
+
+				vmi := libvmi.NewFedora(
+					libvmi.WithInterface(linuxBridgeInterface),
+					libvmi.WithNetwork(linuxBridgeNetwork),
+				)
+
+				vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+				Expect(err).ToNot(HaveOccurred())
+
+				Expect(runSafeCommand(vmiOne, fmt.Sprintf("cat %s | grep %s\n", mtuPath, getPodInterfaceMtu(vmi)))).To(Succeed())
```

Comment:
Retrieve Pod MTU and vmi MTU and use an Equal gomega matcher so it's clear what is failing, failing Greg is not that bad but not as good as an explicit matcher.

Reply:
done